packages/manager/tools/sao-ovh-manager-app/template/src/app.module.js

```diff
@@ -6,6 +6,9 @@ import ovhManagerCore from '@ovh-ux/manager-core';
 import { detach as detachPreloader } from '@ovh-ux/manager-preloader';
 import ngOvhUiRouterLineProgress from '@ovh-ux/ng-ui-router-line-progress';
 import ngUiRouterBreadcrumb from '@ovh-ux/ng-ui-router-breadcrumb';
+import errorPage from './error';
+
+import '@ovh-ux/ui-kit/dist/css/oui.css';
```

Comment:
This `sao-ovh-manager-app` generator defines `@ovh-ux/ui-kit` as a dependency.

I noticed that from the `sao-ovh-manager-module` templates, we are using some `<oui-*` components.

I think that `@ovh-ux/ui-kit` can be defined as a peerDependency when we generate a module.

Is that make sense for you?